Assertion '!empty()' failed.

Hello,

Currently, I am  trying to use coz, however whenever I run it with ANY application with no flags I get the following error

```
/usr/include/c++/14.1.1/bits/basic_string.h:1328: std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::reference std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::front() [with _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>; reference = char&]: Assertion '!empty()' failed.
```

How do fix?

Mabe its something on my side, but i get this error when compiling:
```
cursor.cc: In member function ‘void dwarf::cursor::string(std::string&)’:
cursor.cc:95:27: error: lvalue required as unary ‘&’ operand
   95 |         memmove(&out.c_str(), p, size);
      |                  ~~~~~~~~~^~
```

--- a/dwarf/cursor.cc
+++ b/dwarf/cursor.cc
@@ -92,7 +92,9 @@ cursor::string(std::string &out)
         size_t size;
         const char *p = this->cstr(&size);
         out.resize(size);
-        memmove(&out.front(), p, size);
+        if (size > 0) {
+                memmove(&out.front(), p, size);
+        }
 }
 
 const char *

